src/display/annotation_layer.js

```diff
@@ -240,36 +244,7 @@ class AnnotationElement {
   }
 
   #changeLightness(r, g, b) {
```

Comment:
This private method can now be removed because it's only called in one place in this file, so let's just inline the call to `changeLightness` there to reduce indirection.